Gopkg.toml

```diff
@@ -41,3 +41,7 @@ required = [
 name = "gopkg.in/fsnotify.v1"
```

Comment:
could you split the vendoring commits from the actual code commit?

Reply:
Done